lib/ansible/modules/system/ipset.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Susant Sahani <susant@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ipset
+short_description: Automateds ipset
+description:
+    - Allows you to setup rules to quickly and easily block of setup ip addresses.
+version_added: "2.8"
+options:
+    ipset_name:
+        description:
+            - creating a new "set" of network addresses.
+    hash_type:
+       description: What kind of set
+       choices: [ "hash:net", "hash:ip", "hash:mac", "hash:ip,mac", "hash:net,net", "hash:ip,port",
+                  "hash:net,port", "hash:ip,port,ip", "hash:ip,port,net", "hash:ip,mark", "hash:net,port,net",
+                  "hash:net,iface" ]
+    keys:
+        description:
+            - Space separated list of kesy according to the hash type.
```

Comment:
Typo. "Space separated list of keys according to the hash type."

Reply:
Thanks for the review @jbenden . Updated.